eclass/python-utils-r1.eclass

```diff
@@ -1432,7 +1441,7 @@ epytest() {
 	for x in "${EPYTEST_IGNORE[@]}"; do
```

Comment:
Please update the above `makeopts_jobs` to use `get_makeopts_jobs` instead :)

Reply:
`get_makeopts_jobs` has crappy defaults, so I'd have to inline the `get_nproc` invocation everywhere, that would effectively lead to unconditional performance regression and added complexity. That's what you get when people randomly change stuff without even bothering to understand the original code.